portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,20 +37,44 @@
   private static final String DEFAULT_COMMENTS_COUNT = "5";
   private static final String DEFAULT_PAGE = "1";
   private static final String DEFAULT_COMMENT = "";
+  private static final String DEFAULT_NAME = "";
```

Comment:
For empty string literals, there is no need to define variables. Feel free to remove these.